.github/workflows/qa-main.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         uses: ./
         continue-on-error: true
         with:
+          scannerVersion: 6.1.0.4477
```

Comment:
Why is this change needed? We should still accept blank/empty/null `scannerVersion` to use the default version.